code/modules/projectiles/ammunition/ballistic/pistol.dm

```diff
@@ -21,7 +21,7 @@
 	desc = "A 10mm incendiary bullet casing."
 	projectile_type = /obj/item/projectile/bullet/incendiary/c10mm
 
-// 9mm (Stechkin APS)
+// 9mm (VP78)
```

Comment:
This doesn't do anything but obstuficate what the gun does

Reply:
Reverted anyway